jobclient/python/tests/test_job.py

```diff
@@ -286,9 +308,22 @@ def _check_group(self, job: Job, jobdict: dict):
         else:
             self.assertIsNone(job.group)
 
+    def _check_disk(self, job: Job, jobdict: dict):
+        """Check the disk field of a Job.
+
+        The Job's disk attribute should be either:
+        * The value of `jobdict['disk']` if set,
+        * Otherwise, the value of `jobdict['disk'][0]` if set,
```

Comment:
I don't understand this bullet

Reply:
forgot to update docstring, that bullet is irrelevant and removed now